staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -64,8 +64,8 @@ const legacyAPIServiceName = "v1."
 type ExtraConfig struct {
 	// ProxyClientCert/Key are the client cert used to identify this proxy. Backing APIServices use
```

Comment:
update godoc please

Reply:
hmm this is still true though, right? Or do you want me to explain here that it's used for the dynamic reloading?